resources/multisite_drupal_standard.make

```diff
@@ -1016,9 +1016,9 @@ projects[ec_resp][download][tag] = 2.3.4
 projects[atomium][type] = theme
 projects[atomium][download][type] = git
```

Comment:
Atomium is in d.org as a contrib module.

Reply:
Just for the record:
I just adapted the part requested by the ticket. 
The rest was already like that before and I did not change because we do not have rules for this kind of case.

I will adapt it.